v2/expectation.go

```diff
@@ -93,8 +93,8 @@ func (t *NthTest) Desc() string {
 
 // Do implements Expectation
 func (t *NthTest) Do(ctx context.Context, resp Response) (err error) {
-	root, err := lzjson.Decode(resp.Body())
-	if err != nil {
+	root := lzjson.Decode(resp.Body())
```

Comment:
Thanks a lot for your help!

I knew that would happen. Just didn't have time to do it yet.